lsp-dart.el

```diff
@@ -65,6 +88,28 @@ imported into the current file. Defaults to true"
      ,(expand-file-name (f-join lsp-dart-sdk-dir "bin/snapshots/analysis_server.dart.snapshot"))
      "--lsp")))
 
+(defun lsp-dart--handle-closing-labels (_workspace params)
+  "Closing labels notification handling.
+PARAMS closing labels notification data sent from WORKSPACE."
+  (-let* (((&hash "uri" "labels") params)
+          (buffer (lsp--buffer-for-file (lsp--uri-to-path uri))))
+    (remove-overlays (point-min) (point-max) 'lsp-dart-closing-labels t)
```

Comment:
Sorry for not providing all information in the review. It should be:

```
(when buffer
  (with-current-buffer buffer 
     (remove-overlays ...
```
otherwise, the call will be executed in the context of the current-buffer which is not necessary to be the same as `uri`.

Reply:
It makes sense, Done!

Reply:
Thank you!